documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,90 @@
+3. Comply with all [WCA Competition Policies](wca{documents}).
+4. Be publicly accessible.
+5. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue.
+6. Be held on a single date or on two or more consecutive days.
+7. Have at least one official event.
+8. Have at least one competitor.
+   1. The competition should have at least 12 competitors.
+9. Have at least one designated WCA Delegate in attendance.
+   1. The WCA Delegate for a competition is responsible for:
+      1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
+      2. Sending the competition results to the WCA Results Team within one week of the end of the competition.
+         1. Results should be ready at the end of the last day of competition. Corrections to the results should be available within one week of the competition date.
+         2. All scramble sequences used during a competition must be sent with the results of the competition.
+            1. All scramble sequences must be labeled with the events, rounds, and groups for which they were used. If there are multiple groups on a round, it is not necessary to identify which competitor was in which group.
+         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained permanently.
+      3. Sending corrections of competition results to the WCA Results Team.
+      4. Advising other competition officials when necessary.
+      5. Deciding about disqualifying competitors during the competition.
+      6. Generating, providing, and maintaining the security of the scramble sequences ([WCA Regulation 4b2)](wca{regulations/#4b2})).
+      7. Making a copy of the WCA Regulations available at the competition.
+   2. The WCA Delegate may delegate some of their responsibilities to organizers or other officials.
+
+If WCA Regulations are not correctly observed during a competition, results may be modified at the discretion of the WCA Delegate, WCA Regulations Committee, WCA Results Team, WCA Disciplinary Committee, or the WCA Board.
+
+### Announcement Criteria
+The WCA Delegate must submit the following information when requesting approval of a competition:
+
+1. Competition name, nickname, and ID
+   1. The competition name must abide by the [WCA Competition Name Policy](wca{documents/policies/Competition Name.pdf}).
+   2. The competition nickname and ID must resemble the competition name.
+      1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters).
```

Comment:
```suggestion
       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: World and Continental Championships should use nicknames and IDs that are consistent with past editions.
```

What about this @EdHollingdale?

Reply:
```suggestion
      1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: Groups of competitions may have modified nicknames to ensure consistent nicknames with past and future competitions.
```

Reply:
Not sure if change above is too general? I like having the exception for World and Continental championships explicitly called out, and I think having the consistency with WC_ _ _ _  is important. But I feel we should also allow exceptions for Jacob's example below (Fall/Winter/Summer/Spring competitions) and groups of competitions (e.g. West Coast Cubing Tour Fresno 2018 fits but West Coast Cubing Tour Los Angeles 2018 would not).

Reply:
I'm ok with adding that sentence if others agree! But I would edit it to include IDs too.

Reply:
I agree, the ID should also be covered with this.
```suggestion
      1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
```

Reply:
@thewca/board do you agree with the proposed wording by @EdHollingdale?